tests/core/thermal_conductivity_phase_change_01.cc

```diff
@@ -0,0 +1,94 @@
+/**
+ * @brief Tests the thermal_conductivity_phase_change model.
+ * During a phase change the physical properties (including conductivity) of the
+ * material changes. This interval is defined by [T_solidus,T_liquidus]. This
+ * test verifies that the thermal conductivity is calculated correctly. For
+ * temperature T<T_solidus, the thermal conductivity should be
+ * thermal_conductivity_s. For temperatures T>T_liquidus the thermal
+ * conductivity should be thermal_conductivity_l. For temperature in-between, T
+ * \in [T_solidus,T_liquidus] the thermal conductivity should be liquid_fraction
+ * * thermal_conductivity_l + (1 - liquid_fraction) * thermal_conductivity_s
+ */
+
+// Lethe
+#include <core/parameters.h>
+#include <core/thermal_conductivity_model.h>
+
+// Tests (with common definitions)
+#include <../tests/tests.h>
+
+void
+test()
+{
+  deallog << "Beggining" << std::endl;
+
+  Parameters::PhaseChange phase_change_param;
+  phase_change_param.thermal_conductivity_l = 10;
+  phase_change_param.thermal_conductivity_s = 20;
+  phase_change_param.T_solidus              = 1;
+  phase_change_param.T_liquidus             = 2;
+
+
+  ThermalConductivityPhaseChange thermal_conductivity_model(
```

Comment:
Construct using the whole phase change parameters here instead. Will be simple than passing 4 random doubles

Reply:
done.